examples/units_and_coordinates/stereo_loc_sunpy.py

```diff
@@ -0,0 +1,57 @@
+"""
+============================================
+Getting the location and trajectory of STEREO-A using sunpy
+============================================
+
+This example shows how to get and plot the position of planetary bodies within the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
+"""
+import datetime
+
+import matplotlib.pyplot as plt
+
+from astropy.time import Time
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+##############################################################################
+# You can make use of `~datetime.datetime.now` to pass current date and time.
+# Now can use `get_horizons_coord() <https://docs.sunpy.org/en/stable/generated/api/sunpy.coordinates.get_horizons_coord.html>`__
+# to get the coordinates of STEREO-A.
+
+today = datetime.datetime.now()
+obstime = Time(today)
+aia = get_horizons_coord('STEREO-A', obstime)
+trajectory_coords = get_horizons_coord('STEREO-A', {'start': '2012-11-11', 'stop': '2013-05-11', 'step': '180m'})
```

Comment:
```suggestion
trajectory_coords = get_horizons_coord('STEREO-A', {'start': '2012-11-11', 'stop': '2013-05-11', 'step': '5d'})
```

Reply:
Making it 5d would make the line discontinuous as the interval is large

Reply:
Is that a problem?

Reply:
Yeah, it will be somewhat like this-
![image](https://user-images.githubusercontent.com/64737847/217941332-190ff78e-1b6c-44d8-ae71-461a005b5a0d.png)

Reply:
What's the problem with 180m?

Reply:
Maybe nothing in fact.